stan/math/fwd.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef STAN_MATH_FWD_HPP
+#define STAN_MATH_FWD_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+
+#include <stan/math/fwd/core.hpp>
+#include <stan/math/fwd/meta.hpp>
+#include <stan/math/fwd/vectorize.hpp>
+#include <stan/math/prim/arr.hpp>
+#include <stan/math/prim/mat.hpp>
+#include <stan/math/prim/scal.hpp>
+
+#include <stan/math/fwd/fun.hpp>
+#include <stan/math/fwd/functor.hpp>
```

Comment:
Wouldn't it make more sense to put the fwd headers together?

Reply:
I used the same order as is used in the existing /arr.hpp, /mat.hpp and scal.hpp. See https://github.com/stan-dev/math/blob/develop/stan/math/fwd/arr.hpp, https://github.com/stan-dev/math/blob/develop/stan/math/fwd/scal.hpp, https://github.com/stan-dev/math/blob/develop/stan/math/fwd/mat.hpp

I however dont know the exact reasons behind the order. If you feel that this should be addressed we can discuss this here or on discourse.

Reply:
No big deal, it looked funny, that's all.